db/schema.rb

```diff
@@ -13,11 +13,9 @@
 ActiveRecord::Schema.define(version: 2020_07_10_080945) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "citext"
```

Comment:
What's the reason behind these being removed?

Reply:
```
commit 5209cd7703b503f128b8d1330530618444a21d66
Author: David Mears <david.mears@digital.education.gov.uk>
Date:   Wed Jul 8 12:28:56 2020 +0100

    Remove database extensions

    These were added 13 days ago by tatyree.
    If I drop my database and recreate and migrate, these lines
    no longer exist.
    They most likely were created by tatyree importing a
    production database dump into his local database.
```

Reply:
if your migration is not involved in that, leave it like it is. We have different extensions on our local machine and PaaS postgres service, that's why you end up with changes in that file. Please only commit the changes that are the effect of your migrations. Do not commit changes that are a result of the different databases